gzip/Dockerfile.bug

```diff
@@ -1,8 +1,15 @@
 FROM squareslab/manybugs:gzip
 
 ARG perl
-RUN perlbrew install "perl-${perl}" && \
-    perlbrew use "perl-${perl}"
+RUN wget -O - https://install.perlbrew.pl | bash -f && \
+    source ~/perl5/perlbrew/etc/bashrc && perlbrew init && \
+    echo "source ~/perl5/perlbrew/etc/bashrc" >> ~/.profile && \
+    perlbrew -f install-patchperl && \
+    perlbrew -f install perl-${perl} && \
+    perlbrew switch "perl-${perl}" 
+
+#RUN perlbrew install "perl-${perl}" && \
```

Comment:
It's fine to just remove these lines.

Reply:
Hmmmm, I ran into perlbrew installation problems with the Ubuntu 16.04 update - i couldn't get the original lines to work verbatim. I can try again?

Reply:
If the new lines are functionally equivalent, then we can drop the old ones.

Reply:
There was a perl compatibility issue that I also found - I can't remember the minutiae of this dev.
So how about this: Let me make this change **last** after verifying all other changes to minimize the changeset.

Reply:
So, I was wrong and you were right, @ChrisTimperley.  Good catch. 
The original code **does** work and my change was unnecessary.  I don't know what I might have screwed up when I was creating the original set of dockerfiles/docker images, but c'est la vie.